src/IBI.WZDx/Models/RoadEvents/WorkZones/WorkZoneRoadEvent.cs

```diff
@@ -89,6 +89,12 @@ namespace IBI.WZDx.Models.RoadEvents.WorkZones;
 /// A list of zero or more road restrictions that apply to the roadway segment described by this
 /// road event.
 /// </param>
+/// <param name="ImpactedCdsCurbZones">
+/// A list of references to external CDS Curb Zones impacted by the work zone.</param>
```

Comment:
For consistency with WZDx docs, to help users, please `see href` on "CDS Curb Zones" to https://github.com/openmobilityfoundation/curb-data-specification/tree/main/curbs#curb-zone